src/modules/cpu/kernel/tensor_sum.hpp

```diff
@@ -28,70 +28,70 @@ RppStatus image_sum_u8_u8_host_tensor(Rpp8u *srcPtr,
         Rpp8u *srcPtrChannel;
         srcPtrChannel = srcPtrImage + (roi.xywhROI.xy.y * srcDescPtr->strides.hStride) + (roi.xywhROI.xy.x * layoutParams.bufferMultiplier);
 
-        Rpp32u alignedLength = (bufferLength / 24) * 24;
-        Rpp32u vectorIncrement = 24;
-        Rpp32u vectorIncrementPerChannel = 8;
+        Rpp32u alignedLength = (bufferLength / 48) * 48;
+        Rpp32u vectorIncrement = 48;
+        Rpp32u vectorIncrementPerChannel = 16;
 
-        // Image Sum without fused output-layout toggle (NCHW)
+        // Tensor Sum without fused output-layout toggle (NCHW)
         if ((srcDescPtr->c == 1) && (srcDescPtr->layout == RpptLayout::NCHW))
         {
-            alignedLength = (bufferLength / 8) * 8;
-            vectorIncrement = 8;
-            Rpp64f sum = 0.0;
-            Rpp64f sumAvx[4] = {0.0};
+            alignedLength = (bufferLength / 16) * 16;
```

Comment:
@snehaa8 Pls change this wherever we have powers of 2 - like 8, 16, 32, 64.. 
https://github.com/GPUOpen-ProfessionalCompute-Libraries/rpp/blob/master/src/modules/cpu/kernel/brightness.hpp#L190